src/main/java/com/tagtraum/perf/gcviewer/model/AbstractGCEvent.java

```diff
@@ -662,6 +662,7 @@ public String toString() {
         public static final Type UJL_CMS_CONCURRENT_OLD = new Type("Old", Generation.TENURED, Concurrency.CONCURRENT, GcPattern.GC_MEMORY);
 
         // unified jvm logging g1 event types
+        public static final Type UJL_G1_PAUSE_YOUNG = new Type("pause (G1 Evacuation Pause) (young)", Generation.YOUNG, Concurrency.SERIAL, GcPattern.GC_PAUSE);
```

Comment:
I am quite sure, that this event should not be defined, because support for -XX:+PrintGCCause was introduced years ago (#66). Please delete this line or back it up with a unittest, which fails without this line.

Reply:
Done